db/db.go

```diff
@@ -37,6 +37,7 @@ type hardware interface {
 type template interface {
 	CreateTemplate(ctx context.Context, name string, data string, id uuid.UUID) error
 	GetTemplate(ctx context.Context, fields map[string]string) (string, string, string, error)
+	GetTemplateForWorkflow(ctx context.Context, fields map[string]string) (string, string, string, error)
```

Comment:
I don't want to expose another API if not needed. Can you make `ListTemplates` smarter and remove this call?

Reply:
I have made `GetTemplate` API smarter and removed this new API.

Reply:
Done